master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -138,7 +139,7 @@ We'll define some network policy through the Kubernetes API.  Run the following
 a NetworkPolicy which allows traffic to nginx pods from any pods in the advanced-policy-demo Namespace. 
 
 ```shell
-kubectl create -f - <<EOF 
+cat << EOF | kubectl create -f -
```

Comment:
Sound fine to me.  Changing the calicoctl ones isn't release critical though, so please hold off until after v2.0 :)